docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -0,0 +1,163 @@
+            Selected = "pagination_ic_sel.png"
+        },
+        Name = "Pagination2",
+        Size = new Size(300, 50),
+        BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 0.6f),
+    };
+    ```
+
+2. Use the style to create a Pagination and add it to parent:
+
+    ```cs
+    pagination = new Pagination(style);
+    pagination.IndicatorCount = 5;
+    pagination.SelectedIndex = 2;
+    root.Add(pagination);
+    ```
+
+Following output is generated when the Pagination is created using style:
+
+![Pagination2](./media/Pagination2.png)
+
+## Create with defined styles
+
+You can define a style based on the user experience (UX) and then use this style to create a Pagination.
+
+1. Define a custom style:
+
+    ```cs
+    internal class CustomPaginationStyle : StyleBase
+    {
+        protected override ViewStyle GetViewStyle()
+        {
+            PaginationStyle style = new PaginationStyle
+            {
+                IndicatorSize = new Size(15, 15),
+                IndicatorSpacing = 20,
+                IndicatorImageUrl = new Selector<string>
+                {
+                    Normal = "pagination_ic_nor.png",
+                    Selected = "pagination_ic_sel.png"
+                },
+                Name = "Pagination2",
+                Size = new Size(300, 50),
+                BackgroundColor = new Color(1.0f, 1.0f, 1.0f, 0.6f),
+            };
+            return style;
+        }
+    }
+    ```
+
+2. Register your custom style:
+
+    ```cs
+    StyleManager.Instance.RegisterStyle("CustomPagination", null, typeof(YourNameSpace.CustomPaginationStyle));
+    ```
+
+3. Use your custom style to create a Pagination instance:
+
+    ```cs
+    pagination = new Pagination("CustomPagination");
+    pagination.IndicatorCount = 5;
+    pagination.SelectedIndex = 2;
+    root.Add(pagination);
+    ```
+
+Following output is generated when the Pagination is created using the defined style:
+
+![Pagination2](./media/Pagination2.png)
+
+## Responding to window key event
+
+When you press "left", the currently active page will change to previous page.
+When you press "right", the currently active page will change to next page.
+You can declare the window key event as follows:
+
+```cs
+Window window = NUIApplication.GetDefaultWindow();
+window.KeyEvent += Window_KeyEvent;
+```
+
+```cs
+private void Window_KeyEvent(object sender, Window.KeyEventArgs e)
+{
+    if (e.Key.State == Key.StateType.Down)
+    {
+        if (e.Key.KeyPressedName == "Left")
+        {
+            if (pagination.SelectedIndex > 0)
+            {
+                pagination.SelectedIndex = pagination.SelectedIndex - 1;
+            }
+        }
+        else if (e.Key.KeyPressedName == "Right")
+        {
+            if (pagination.SelectedIndex < pagination.IndicatorCount - 1)
+            {
+                pagination.SelectedIndex = pagination.SelectedIndex + 1;
+            }
+        }
+    }
+}
+```
+
+## Related Information
```

Comment:
Since we are following sentence case for all the headings. Suggest changing "Related Information" to "Related information".

@mijin85cho, in all other files under docs/application/dotnet/guides/nui/nui-components/ folder , "Related Information" is in title case. Please clarify shall we change to sentence case or leave it as it is?

Reply:
fixed

Reply:
@surbhig12 As far as I remember, we asked you to check whole headings and update as like https://github.com/Samsung/tizen-docs/pull/1036. However it is delayed because of homeworking of SIR-B and other issues.

So for now we are using sentence cases to the heading in newly added pages or to the pages on review. Please keep using sentence case. For other pages, I hope SRI-B to update them as we asked before, but it has low priority.